ledger/src/blockstore_db.rs

```diff
@@ -526,6 +541,21 @@ impl Rocks {
             Err(e) => Err(BlockstoreError::RocksDb(e)),
         }
     }
+
+    fn print_error_handling_instruction(rocks_error: &rocksdb::Error) {
+        let error_msg = format!("{:?}", rocks_error);
+        if error_msg.contains("Corruption:") {
```

Comment:
i think rocksdb::Error::Kind can be used?

https://docs.rs/rocksdb/latest/rocksdb/struct.Error.html#method.kind

https://docs.rs/rocksdb/latest/rocksdb/enum.ErrorKind.html#variant.Corruption

Reply:
Oh, oh, I didn't know this is available in rust-rocksdb.  Thanks for pointing it out!